velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -408,4 +408,47 @@ struct DayOfYearFunction {
     result = getDayOfYear(getDateTime(date));
   }
 };
+
+template <typename T>
+struct DateFormatFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  const date::time_zone* sessionTimeZone_ = nullptr;
```

Comment:
Make these variables private and move them to the end of the struct.

sessionTimeZone_{nullptr};
isConstFormat_{false};